.github/workflows/metadata-ingestion.yml

```diff
@@ -62,8 +63,7 @@ jobs:
           pip install pip -U # only 22.2 and above contains the --dry-run flag
 
           # only the last line of the output is the packages that will be installed
-          pip install --dry-run -e .  | tail -n 1 > /tmp/would_be_installed.txt
-          echo ${{ matrix.extraPythonRequirement }} --dry-run >> /tmp/would_be_installed.txt
+          pip install --dry-run -e .  ${{ matrix.extraPythonRequirement }} | tail -n 1 > /tmp/would_be_installed.txt
```

Comment:
```suggestion
          pip install --dry-run -e .[dev]  ${{ matrix.extraPythonRequirement }} | tail -n 1 > /tmp/would_be_installed.txt
```

Reply:
we could also add [integration-tests], but that's not used by all tests in the matrix